doc/devel/contributing.rst

```diff
@@ -115,28 +100,15 @@ reflected the next time you import the library.  If you change the
 C-extension source (which might happen if you change branches) you
 will need to run ::
 
-   python setup.py build
+   python setup.py build_ext --inplace
 
 or re-run ``python -mpip install -ve .``.
 
-Alternatively, if you do ::
-
-   python -mpip install -v .
-
-all of the files will be copied to the installation directory however,
-you will have to rerun this command every time the source is changed.
-Additionally you will need to copy :file:`setup.cfg.template` to
-:file:`setup.cfg` and edit it to contain ::
-
```

Comment:
So is this not needed anymore?

Reply:
Not if you always use an editable install.

Reply:
Are pip and setuptools now smart enough to handle changes to extension code in editable mode?  My sense is that editable mode works great for changes to existing python modules, but not beyond that.

Reply:
C files are mentioned above and still have the note "rebuild if you edit one of them".